prog/dftb+/api/mm/mmapi.F90

```diff
@@ -391,6 +391,13 @@ subroutine TDftbPlus_setExternalPotential(this, atomPot, potGrad)
 
     call this%checkInit()
 
+    if (allocated(this%main%solvation)) then
+      if (this%main%solvation%isEFieldModified()) then
+        write(stdOut, "(A)") "Error: External fields currently unsupported for this solvent model"
+        stop
```

Comment:
There is still the question if we really should stop here? I don't think it is a good idea to stop in the API and return with exit code 0. If we really have to kill the API user, can we at least return with a non-zero exit code, please?

Reply:
@awvwgk are you asking for just an "error stop", or should we discuss meaningful error codes?
This brings up the need to have proper error code return from the API.

Reply:
Exactly, I'm just asking for some non-zero exit code on an error, `error stop` will do.